src/Preferences.cc

```diff
@@ -92,9 +92,9 @@ class SettingsReader : public Settings::SettingsVisitor
 
 	std::string key = entry.category() + "/" + entry.name();
 	std::string value = settings.value(QString::fromStdString(key)).toString().toStdString();
-	const Value v = getValue(entry, value);
+	Value v = getValue(entry, value);
 	PRINTDB("SettingsReader R: %s = '%s' => '%s'", key.c_str() % value.c_str() % v.toString());
-	s->set(entry, v);
+	s->set(entry, std::move(v));
```

Comment:
As mentioned elsewhere, we can push this `move()` into the `set()` function and let the object be copied when passing by value. If we, e.g. somewhere else, want to pass a Value literal, that will be move constructed and no copy is made.

Reply:
I can change `set` signature to pass by value, but it wouldn't make much difference from this context.  Since one of the main things I changed was to delete the copy constructor on `Value`, and `v` is already an lvalue, I would still need to either move or clone it in this call.  If the `PRINTDB` in the line above weren't using v, I could I could do `s->set(entry, getValue(entry, value));`.  (OK I guess regardless of `PRINTDB` I could do that but it would be a redundant call to `getValue` instead of calling `std::move` *shrug*).

Reply:
Deleting the copy constructor was really eye-opening and taught me a lot.  I like the idea of having to explicitly clone when needed rather than having the compiler automatically do it, it makes you more aware of where things happen. 
 Then again Values should still be fairly cheap to copy I think, so if you think it would be better to just un-delete copy ctor, we could do that too.  I was originally trying to work towards having no copies at all somehow(and being able to use `unique_ptr` rather than `shared_ptr`), so seeing every clone operation was helpful, but it seems that eradicating copies entirely is not feasible with the way OpenSCAD works.

Reply:
I didn't really think about the deleted copy constructor. Perhaps let's revisit this once I've reviewed the rest..